File: api/src/main/java/fr/mipih/rh/testcandidats/dtos/SandboxDto.java
package fr.mipih.rh.testcandidats.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class SandboxDto {

    private Long id;
    private String src;
    private String niveau;
    private String technologie;
    private String nom;
    private String consigne;

    public SandboxDto(long l) {
    }
}

File: front-end/src/app/share/dtos/sandbox/sandbox-dto.ts
import { Niveau } from "../../enums/niveau.enum";
import { Technologie } from "../../enums/technologie.enum";

export interface SandboxDto {
  id?: number;
  src: string;
  consigne: string;
  nom: string;
  niveau: Niveau;
  technologie: Technologie;
}
